resources/gemini-1b/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "remote" {
+    hostname      = "app.terraform.io"
+    organization  = "subspace-sre"
```

Comment:
We can create another organisation for production usage. subspace-sre was created mostly for testing.

Reply:
is there a difference between this org in comparision to new one? If not,  I think we can delete the testing repos and can create a new one for testing. wdyt ?

Reply:
There's no difference. Sure lets create another one for testing. Also I planning to use them for stages. That can be run straight out of U by click of the button